index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Monicah capstone project</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/a6627283f4.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <section class="main-page">
+        <nav class="top-bg">
+            <div>
+            
+                <ul class="top">
+                    <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+                    <a href=""><i class="fa-brands fa-twitter"></i></a>
+                  <li>Engish</li>
+                  <li>My page</li>
+                  <li>Logout</li>
+                </ul>
+            </div>
+        </nav>
+        <header class="header" >
+            <div class="lg-logo">
+                <a href="homepage.html">
+                  <img src="Images/compost_FILL.svg" alt="logo img">
+                </a>
+            <a  class="logo" href="">Eco Tourism<br> Kenya 2022 </a>
+            </div>
+            
+            <nav class="nav-1">
+    
+                <ul class="list">
+                
+                    <li class="list-li bar-list"><a href="homepage.html">About</a></li>
+                    <li class="bar-list"><a>program</a></li>
+                    <li class="bar-list"><a>Join</a></li>
+                    <li class="bar-list"><a>Sponsor</a></li>
+                    <li class="bar-list"><a>News</a></li>
+                    <li class="btn bar-list"><a>CC Captain</a></li>
+                </ul>
+          
+                     
+             <div class="menu"> 
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+             </div>
+            </nav>  
+    </header>           
+        <div class="cover">
+          <div>
+               <h2 class="first">"Hello!Sharing Awarness"</h2>
+               <h1 class="second">ISIOLO DROUGHT <br> RELIEF RESPONSE <br> 2022</h1>
+          </div>
+               <div>
+                  <p class="second-pr">The prolonged drought has brought food shortage to more <br>
+                than 3 million people in arid and semi-arid areas in Kenya.<br>
+            The GreenTour Kenya team has partnered with Kenya Red Cross to raise funds for families in Isiolo.<br>
+             We are targeting 300 families.</p>
+               </div>
+               <div>
```

Comment:
- [ ] Dear @monicah79, As the previous code reviewer mentioned please wrap the articles inside an `article `tag. this is for semantic reasons.